modules/serverless-deploy-func-kn.adoc

```diff
@@ -6,14 +6,12 @@
 [id="serverless-deploy-func-kn_{context}"]
 = Deploying functions
 
-You can deploy a function to your cluster as a Knative service by using the `kn func deploy` command.
-
-If the targeted function is already deployed, it is updated with a new container image that is pushed to a container image registry, and the Knative service is updated.
+You can deploy a function to your cluster as a Knative service by using the `kn func deploy` command. If the targeted function is already deployed, it is updated with a new container image that is pushed to a container image registry, and the Knative service is updated.
```

Comment:
Do we use the term "service" generically like this? Or should it be the specific Knative `Service` primitive? Genuine question. E.g.
```suggestion
You can deploy a function to your cluster as a Knative `Service` by using the `kn func deploy` command. If the targeted function is already deployed, it is updated with a new container image that is pushed to a container image registry, and the Knative `Service` is updated.
```

Reply:
We only use `Service` if we're referring to a specific "`Service` object". For general use it's "Knative service". Same sort of thing for any resources like this, e.g. general use is "config map", "channel", etc unless we're referring to a specific example of an object, e.g. "`ConfigMap` object", "`Channel` object", etc.. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Worth noting that upstream doc guidelines for this are different to downstream. I'd like to see it standardized, but it will not be changing for downstream, and there was push back on changing it upstream 🤷🏻